Tests/Admin/AdminHelperTest.php

```diff
@@ -18,6 +18,9 @@
 
 class AdminHelperTest extends \PHPUnit_Framework_TestCase
 {
+    /**
+     * @covers Sonata\AdminBundle\Admin\AdminHelper::getChildFormBuilder
+     */
```

Comment:
> Oh ok, so @covers makes sure the coverage doesn't say this method covers something else?

exactly 👍  so we could get a nice overview where we need to invest time/work/code. Currently i have a collegue who wants to write tests and asked me what he should test ;-)

> But what if the indirectly covered method is a protected or private method ?

seems to be no problem